hogwarts/scss/_article.scss

```diff
@@ -125,9 +125,50 @@ article {
 
     }
   }
+
+  // ipython stuff
+  .prompt {
+      display: none;
+  }
+
+  .output_png {
+    width: 100%;
+    text-align:center;
+    padding: 1rem;
+  }
+
+  .output_text {
+    pre {
+      padding: 0;
+      font-size: 1rem;
+    }
+  }
+
+  .code_cell {
+    pre {
+      font-size: 1rem;
+    }
```

Comment:
you should put that in the top level article actually, we also want our own code sample to be that size